src/karma.conf.js

```diff
@@ -23,10 +23,10 @@ module.exports = function(config) {
             ],
             check: {
                 global: {
-                    statements: 99.73,
+                    statements: 99.72,
                     branches: 99.28, // always keep it 0.02% below local coverage
                     functions: 99.78,
-                    lines: 99.73,
+                    lines: 99.72,
```

Comment:
eeeeh, diminishing !

Reply:
less code = less percentage ≠≠≠ less coverage, as usual !

Reply:
On a enlevé du code ?
Bah, quand on enlève du code alors il faut se dire qu'il faut aller en couvrir un autre ailleurs :p doit-ça-sonne-bon (does that sound good ?) ?

Reply:
Certes mais il est pas dans les fichiers celui-là :/ (enfin, pas en entier haha, faut le recalculer)

Reply:
bah on a notre script de coverage qui s'en occupe de ça